icu4j/main/classes/core/src/com/ibm/icu/text/DecimalFormat.java

```diff
@@ -2015,10 +2015,10 @@ public synchronized void setMinimumGroupingDigits(int number) {
   }
 
   /**
-   * {@icu} Constant for {@link #setMinimumGroupingDigits()} to specify display
+   * {@icu} Constant for {@link #setMinimumGroupingDigits(int)} to specify display
    * grouping using the default strategy for all locales.
    *
-   * @see #setMinimumGroupingDigits
+   * @see #setMinimumGroupingDigits(int)
```

Comment:
this was broken by something else… I fix it here so that the javadoc build passes.